domain/src/DomainStorage.php

```diff
@@ -82,6 +82,8 @@ public function loadDefaultId() {
 
   /**
    * {@inheritdoc}
+   *
+   * @return \Drupal\domain\DomainInterface|NULL
```

Comment:
these returns should go on the interface

Reply:
Thx for looking!
I've just reviewed this, and these are serving a useful purpose: the interface does have these declarations, but the class {@inheritdoc} is not inheriting the interface but the base class doc, which has EntityInterface.

The net result is that within code manipulating class object refs (e.g. within domain module) these functions return EntityInterface objects, not DomainInterface objects.

I've pushed an update which works for me: let me know if there's a better way.